### PR TITLE
DSNP Annoucing Spec

### PR DESCRIPTION
**Pull request by wilwade**
Note
=======
This is a alternative version of https://github.com/LibertyDSNP/spec/pull/8 and https://github.com/LibertyDSNP/spec/pull/11.
The versions cannot all be merged.

Problem
=======
Users of DSNP need a way to publish messages in a standard way
[link to Pivotal Tracker #23177049245](https://www.pivotaltracker.com/story/show/23177049245)

Solution
========

A separate interface for creating an DSNPBatch event that announces a file that contains the list of new messages.

Change summary:
---------------
* Started a new spec section DSNP Announce
* Added to the sidebar
* Referenced DSNPMessage
* DSNPAnnounce Interface
* DSNPBatch event

### CODE REVIEW

File: pages/DSNP/Messages/Overview.md [hunk 1]
```diff
@@ -65,94 +64,112 @@ For details on how messages are serialized, see [DSNP Message Serialization](/DS
...
 
 #### Drop
 a dead drop message
 
-| field | description | type
-|-------|-------------| ---|
-| ddid | dead drop id |  bytes
-| uri  | content uri  |  bytes
+| dsnpData field | description | type |
+| ------------- |------------- | ---- |
```
Comment (shannonwells) [inline pages/DSNP/Messages/Overview.md:89]: I learned recently you just need three dashes between pipes to cause it to be rendered as a table.

**Review by shannonwells — COMMENTED** [review]
Will this be irrespective of whether routes are used?

File: pages/DSNP/Messages/Overview.md (hunk 1 above)
Comment (wilwade) [reply pages/DSNP/Messages/Overview.md:89]: @shannonwells I wonder if we want a lint formatter to do the table formatting for us... Make them look right every time. Or I am also ok with a standard of just three `---` and spaces before and after. So like:
```markdown
| title | spaces around |
| --- | --- |
| content | also has spaces around |
```

**Comment by wilwade** [comment]
@shannonwells Re: Routes. That is correct. This is independent of routes, and routes should be re-evaluated in the context of the batch file.

**Review by shannonwells — APPROVED** [review]
ell gee tee em

**Pull request merged**